python/pylibcudf/pylibcudf/nvtext/tokenize.pyx

```diff
@@ -0,0 +1,286 @@
+        String scalar used to separate each string into tokens
+
+    Returns
+    -------
+    Column
+        New column of token counts
+    """
+    cdef unique_ptr[column] c_result
+
+    if delimiter is None:
+        delimiter = Scalar.from_libcudf(
+            cpp_make_string_scalar("".encode())
+        )
+
+    with nogil:
+        c_result = cpp_count_tokens(
+            input.view(),
+            dereference(<const string_scalar*>delimiter.c_obj.get()),
+        )
+
+    return Column.from_libcudf(move(c_result))
+
+cpdef Column count_tokens_column(Column input, Column delimiters):
+    """
+    Returns the number of tokens in each string of a strings column
+    using multiple strings as delimiters.
+
+    For details, see cpp:func:`cudf::nvtext::count_tokens`
+
+    Parameters
+    ----------
+    input : Column
+        Strings column to count tokens
+    delimiters : Column
+        Strings column used to separate
+        each string into tokens
+
+    Returns
+    -------
+    Column
+        New column of token counts
+    """
+    cdef unique_ptr[column] c_result
+
+    with nogil:
+        c_result = cpp_count_tokens(
+            input.view(),
+            delimiters.view(),
+        )
+
+    return Column.from_libcudf(move(c_result))
+
+cpdef Column character_tokenize(Column input):
+    """
+    Returns a single column of strings by converting
+    each character to a string.
+
+    For details, see cpp:func:`cudf::nvtext::character_tokens`
+
+    Parameters
+    ----------
+    input : Column
+        Strings column to tokenize
+
+    Returns
+    -------
+    Column
+        New strings columns of tokens
+    """
+    cdef unique_ptr[column] c_result
+    with nogil:
+        c_result = cpp_character_tokenize(input.view())
+
+    return Column.from_libcudf(move(c_result))
+
+cpdef Column detokenize(
+    Column input,
+    Column row_indices,
+    Scalar separator=None
+):
+    """
+    Creates a strings column from a strings column of tokens
+    and an associated column of row ids.
+
+    For details, see cpp:func:`cudf::nvtext::detokenize`
+
+    Parameters
+    ----------
+    input : Column
+        Strings column to detokenize
+    row_indices : Column
+        The relative output row index assigned
+        for each token in the input column
+    separator : Scalar
+        String to append after concatenating
+        each token to the proper output row
+
+    Returns
+    -------
+    Column
+        New strings columns of tokens
+    """
+    cdef unique_ptr[column] c_result
+
+    if separator is None:
+        separator = Scalar.from_libcudf(
+            cpp_make_string_scalar(" ".encode())
+        )
+
+    with nogil:
+        c_result = cpp_detokenize(
+            input.view(),
+            row_indices.view(),
+            dereference(<const string_scalar*>separator.c_obj.get())
+        )
+
+    return Column.from_libcudf(move(c_result))
+
+cpdef TokenizeVocabulary load_vocabulary(Column input):
```

Comment:
Unfortunately I've been leaving you the same feedback on all the nvtext PRs at just the wrong intervals here since you had them all open first, sorry! But I think the same thing I said on the other PRs applies here too, let's get rid of the function and just use the class constructor.